components/tests/ui/testcases/web/search_annotation_test.txt

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Documentation     Tests searching for file annotations
+
+Resource          ../../resources/web/annotation.txt
+Resource          ../../resources/config.txt
+Resource          ../../resources/web/login.txt
+Resource          ../../resources/web/tree.txt
+
+Suite Setup         User "${USERNAME}" logs in with password "${PASSWORD}"
+
+*** Variables ***
+
+${commentText}          Robot test adding this comment
+${fileName}             robot_file_annotation2.txt
+${SEARCH URL}           ${WELCOME URL}search/
+
+*** Test Cases ***
+
+Test Search Results File Annotations
+
+    Select Experimenter
+    ${sId_One}=                                 Create Screen       robot file annotations_1
+
+    # Annotate single Screen
+    # NB: Attachments panel may be open from previous test. Try opening 'closed' panel in case...
+    Run Keyword And Ignore Error                Click Element  xpath=//h1[@data-name='attachments'][contains(@class, 'closed')]
```

Comment:
Can probably replace this line and the comment line above now, since this we won't have variable state from previous test.
Instead, we will always need to expand the annotations panel with
```
Click Element                               xpath=//h1[@data-name='attachments']
```